gradle/build-logic/src/main/kotlin/conventions.base.gradle.kts

```diff
@@ -4,6 +4,7 @@ plugins {
     id("org.incendo.cloud-build-logic")
     id("org.incendo.cloud-build-logic.spotless")
     id("org.incendo.cloud-build-logic.errorprone")
+    id("org.incendo.cloud-build-logic.javadoc-links")
```

Comment:
why was this added here?

Reply:
oh, forgot to remove it. leftover from when I had to exclude brig from javadocs when I was including brig as an api dependency